controllers/utils.go

```diff
@@ -0,0 +1,68 @@
+/*
+Copyright 2020 The Flux authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package controllers
+
+import "strings"
+
+// parseApplyOutput extracts the objects and the action
+// performed by kubectl (created, configured, unchanged)
```

Comment:
Can you add an example of the format of (a line from) the apply output, that would help me to better understand how it is parsed.